components/common/EventPopup/AcmEventHeads.tsx

```diff
@@ -87,55 +86,34 @@ export default function AcmEventHeads({teamTitle, pills, setPills, setEmptyError
     }
 
     return (
-        <div style={{
-            display: "flex",
-            margin: "30px",
-        }}>
-            <div style={{
-                width: "300px"
-            }}>
-                <div style={{
-                    width:"300px",
-                    padding:"20px"
-                }}>
-                    <CustomTextField
-                        type="text"
-                        onChange={ handleNameSearchChange }
-                        value={nameSearchValue}
-                        label={`Search ${teamTitle}`}
-                        fullWidth={true}
-                        variant="filled"
-                    />
+        <div className={styles['acm-event-heads-container']}>
+            <div className={styles.pillInputDiv}>
+                <PillContainer handleDelete={handleDelete} pills={pills} />
+                <input
+                    type="text"
+                    onChange={handleNameSearchChange}
+                    value={nameSearchValue}
+                    className={styles.pillInput}
+                    placeholder={`Search ${teamTitle}*`}
+                />
+            </div>
 
-                    <div>
-                        {
-                            searchResult.map((pill: Pill) =>
-                                <div
-                                    onClick={() => addUserToPills(pill)}
-                                    key={pill.email}
-                                    style={{
-                                        height: "40px",
-                                        borderRadius: "10px",
-                                        padding: "5px"
-                                    }}
-                                >
-                                    {pill.name}
-                                    <p
-                                        style={{
-                                            color:"grey",
-                                            fontSize:"small",
-                                        }}
-                                    >
-                                        {pill.email}
-                                    </p>
-                                </div>)
-                        }
-                    </div>
+            <div>
+                <div className={styles['pills-container']}>
+                    {searchResult.map((pill: Pill) =>
+                        <div
+                            onClick={() => addUserToPills(pill)}
+                            key={pill.email}
+                            className={styles['pill']}
```

Comment:
can you make the drop down look a little better 
<img width="283" alt="image" src="https://github.com/PEC-CSS/website/assets/76965050/61be90ab-fa07-4bb1-b183-141e8d2902fd">
maybe something like this (of course without the images)
<img width="447" alt="image" src="https://github.com/PEC-CSS/website/assets/76965050/e763a350-d482-44ef-b69b-0eb15abe715a">